erica_snider/rails/DojoSecrets/app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class LikesController < ApplicationController
+  def create
+    like = Like.new(user: current_user, secret: Secret.find(like_params[:secret_id]))
+    if not like.save
```

Comment:
Usually I don't recommend using not unless there is a specific reason to use it.